### ISSUE TITLE
[Publications] Stakeholder Kickoff: Data Futures

### ISSUE THREAD

**Issue by sabrinang** [mozilla/foundation.mozilla.org#4946]
Hold individual stakeholder interviews to gather any remaining requirements and inform them we are starting with overall plans. 

Data Stewardship Stakeholder: Kasia?

**Comment by sabrinang** [comment]
Reached out to Kasia on slack to inform her I'm looking into content on https://github.com/mozilla/foundation.mozilla.org/issues/4829 and to send content my way. Scheduled a chat for Thursday (her cal is pretty booked before then).

**Comment by sabrinang** [comment]
I chatted with Kasia and Phil/Eeva/Mehan also joined the call earlier today.

We clarified we are focusing on the publication template. I can refer to their mockup doc for feature priorities and inspiration/benchmarks Kasia to include there or send my way. We’ve discussed their project so I have more context of the content. We went over some content feature ideas and considerations for flexible usage. They’ve mentioned their site is planned to be launched last week of August/early September and ideally they can publish with this nicer template (probably tight for time but they have mockups from TaÍs to work with too).

We touched upon context for the future of IHR needs as well since Eeva could join our call so that was insightful too. 

I will be keep Kasia in the loop for this to check-in that our content types works for their purposes.
